src/dex-ui/pages/dao/RecentProposals.tsx

```diff
@@ -0,0 +1,58 @@
+import { ErrorLayout, LoadingSpinnerLayout, NotFound } from "@layouts";
+import { Proposal } from "@hooks";
+import { Paths } from "@routes";
+import { replaceLastRoute } from "@utils";
+import { isEmpty, isNotNil } from "ramda";
+import { useLocation, useNavigate } from "react-router-dom";
+import { ProposalCard } from "./ProposalCard";
+import { Color, TransactionIcon } from "@dex-ui-components";
+import { DAO } from "@services";
+import { Flex, Text } from "@chakra-ui/react";
+
+interface RecentProposalsProps {
+  proposals: Proposal[] | undefined;
+  dao: DAO;
+  isSuccess: boolean;
+  isLoading: boolean;
+  isError: boolean;
+  error: any;
+}
+
+export function RecentProposals(props: RecentProposalsProps) {
+  const navigate = useNavigate();
+  const location = useLocation();
+  const { proposals, dao, isSuccess, isLoading, isError, error } = props;
+  function handleClickCreateProposal() {
+    navigate(replaceLastRoute(location.pathname, Paths.DAOs.CreateDAOProposal));
+  }
+
+  if (isError) {
+    return <ErrorLayout message={error?.message} />;
+  }
+
+  if (isLoading) {
+    return <LoadingSpinnerLayout />;
+  }
+
+  if (isSuccess && isNotNil(proposals) && !isEmpty(proposals)) {
+    return (
+      <Flex gap="4" direction="column">
+        <Text textStyle="h4 medium">Recent Proposals</Text>
+        <Flex direction="column" gap="2" minHeight="300px">
+          {proposals.map((transaction, index) => (
+            <ProposalCard proposal={transaction} dao={dao} key={index} />
+          ))}
+        </Flex>
+      </Flex>
+    );
+  }
+
+  return (
```

Comment:
The "Recent Proposal" header should be present when the Not Found message is displayed.

Current UI (missing Recent Proposal section header):
![Screen Shot 2023-07-05 at 1 36 39 PM](https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/54907098/75a4cfee-d3c5-4019-8652-e27d96cf582e)

Expected:
![Screen Shot 2023-07-05 at 1 39 54 PM](https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/54907098/7c47a4b9-2317-406e-8bfc-fd3154cf8652)

Reply:
The UI design and experience for this section should match across all DAO dashboards.

Reply:
1. Added the missing heading in the Recent proposals. 
2. Using same component for both the places.

Reply:
<img width="1536" alt="Screenshot 2023-07-06 at 4 40 50 PM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/132666177/ee20c57d-3457-402c-94e9-35ad3cf900e0">
<img width="989" alt="Screenshot 2023-07-06 at 4 41 39 PM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/132666177/1867783a-8522-4a11-a2c7-fd2a631af289">